include/jemalloc/internal/emap.h

```diff
@@ -230,16 +231,53 @@ emap_edata_lookup(tsdn_t *tsdn, emap_t *emap, const void *ptr) {
 	return rtree_read(tsdn, &emap->rtree, rtree_ctx, (uintptr_t)ptr).edata;
 }
 
+JEMALLOC_ALWAYS_INLINE void
+emap_alloc_ctx_set(emap_alloc_ctx_t *alloc_ctx, szind_t szind, bool slab,
+    size_t usize) {
+	alloc_ctx->szind = szind;
+	alloc_ctx->slab = slab;
+	/*
+	 * When config_limit_usize_gap disabled, alloc_ctx->usize
+	 * should not be accessed.
```

Comment:
nit: assert this this way -- 
```
if (config_limit_usize_gap) {
  ...
} else if (config_debug) {
  alloc_ctx->usize = SC_LARGE_MAXCLASS + 1;
}
```
Then in the getter, assert(usize <= SC_LARGE_MAXCLASS)